react-app/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "build": "tsc && NODE_NO_WARNINGS=1 vite build",
-    "build:dev": "VITE_DAY_VARIANT=developer npm run build",
+    "build:dev": "NODE_OPTIONS='--max-old-space-size=8192' VITE_DAY_VARIANT=developer npm run build",
```

Comment:
With more and more calendars and data, I had to increase the memory here, otherwise I couldn't build the site anymore.